hack/demo.sh

```diff
@@ -48,10 +48,11 @@ spec:
   configuration:
     exclude:
     - not_useful
-    - test:conftest-clair
   sources:
-    - name: Default EC policy
-      policy:
+  - data:
+    - github.com/hacbs-contract/ec-policies.git//data
```

Comment:
This should work now if you want to use the bundle here also.

```suggestion
    - quay.io/hacbs-contract/ec-policy-data:latest
```

Reply:
(Delete my other comment about yaml indenting. It's already inconsistent I think, so don't worry it.)

Reply:
Changed to quay.io/hacbs-contract/ec-policy-data:latest.And the bundle works. Thanks!